test/systemtest/utils.py

```diff
@@ -426,19 +426,15 @@ def wait_func():
             file_object.seek(0)
 
     while True:
-        i = 0
         for file_object in file_objects:
-            i = 0
             for line in file_object:
-                i += 1
-
                 m = match_line(line.rstrip(), pattern, filter_func)
                 if m is not None:
                     return m
 
         # Check if we exceed the timeout. Do so only every 100 lines to reduce
         # the performance impact.
-        if timeout is not None and i % 100 == 99 and time.time() >= t_end:
+        if timeout is not None and time.time() >= t_end:
```

Comment:
Did you intend to remove the "1 in 100" bit here? If so, you probably want to change the comment above.

Reply:
Yeah, that's intentional, as it doesn't work well (aka at all) with the more flexible wait function we have today (and it was broken anyways). 

I forgot to remove the comment, fixed.